packages/typespec-autorest/src/openapi.ts

```diff
@@ -2414,6 +2432,7 @@ export async function getOpenAPIForService(
     typespecType: Model,
     context: SchemaContext,
     namespace?: Namespace,
+    property?: ModelProperty,
```

Comment:
Feels like pasing the property around is a bit messy. Can't we just override whatever is set on `x-ms-identifiers` when resolving the property if it has `@identifiers` decorator set